open-authoring/OpenAuthoringModalContainer.tsx

```diff
@@ -1,23 +1,22 @@
 import { useEffect, useState } from 'react'
 import { ActionableModal } from '../components/ui'
-import { enterEditMode } from './authFlow'
+import { startAuthFlow } from './authFlow'
 import { useOpenAuthoring } from '../components/layout/OpenAuthoring'
 import OpenAuthoringContextualErrorUI from './OpenAuthoringContextualErrorUI'
+import getErrorUIFrom from './error-interpreter'
 
 interface Props {
-  error?: OpenAuthoringContextualErrorUI
-}
-
-export enum Actions {
-  authFlow,
-  refresh,
-  doNothing
+  openAuthoringErrorUI?: OpenAuthoringContextualErrorUI
+  openAuthoringError?
 }
 
 
-export const OpenAuthoringModalContainer = ({ error }: Props) => {
+export const OpenAuthoringModalContainer = ({
+  openAuthoringErrorUI,
+  openAuthoringError,
+}: Props) => {
   const [authPopupDisplayed, setAuthPopupDisplayed] = useState(false)
-  const [statefulError, setStateFullError] = useState(error)
+  const [statefulOpenAuthoringErrorUI, setStateFullOpenAuthoringErrorUI] = useState(openAuthoringErrorUI)
```

Comment:
I'd remove "StateFull" from the name here (same with the variables with contextual in the name). If the behavior changes and that error is passed as a prop, we shouldn't have to rename it.